crates/ruff_cli/src/lib.rs

```diff
@@ -117,11 +119,38 @@ quoting the executed command, along with the relevant file contents and `pyproje
             shell.generate(&mut Args::command(), &mut io::stdout());
         }
         Command::Check(args) => return check(args, log_level),
+        Command::Format { files } => return format(&files),
     }
 
     Ok(ExitStatus::Success)
 }
 
+fn format(files: &[PathBuf]) -> Result<ExitStatus> {
+    warn!("This function is unfinished, broken and for internal use only!");
```

Comment:
Nit: you could use `warn_user_once!` here.

Reply:
Going to bikeshed the wording to be consistent in tone with other user-facing copy, maybe like: "`ruff format` is a work-in-progress, subject to change at any time, and intended for internal use only."?

Reply:
that's neat!

Reply:
that's a much nicer phrasing, thanks!